0003/README.md

```diff
@@ -1,19 +1,19 @@
 ---
-name: Messaging IPC
-shortname: 3/MIPC
+name: Inter-Service Messaging
+shortname: 3/ISM
 status: draft
-editor: Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
-contributors: Volodymir Tkhir <vtkhir@cybervisiontech.com>
+editor: Andrew Kokhanovskyi <ak@kaaiot.io>
+contributors: Volodymir Tkhir <vtkhir@kaaiot.io>
 ---
 
 <!-- toc -->
 
+
 ## Introduction
 
-Kaa Banana Beach release uses REST / HTTP API and NATS-based messaging for communication between microservices.
-This document defines the general design principle of the latter type of messaging.
+This document defines the general design recommendations for the NATS-based inter-service messaging protocols for Kaa Banana Beach.
 
-Messaging IPC is classified into two categories:
+NATS-based messaging is classified into two categories:
```

Comment:
It would be good to introduce NATS first.

Reply:
I'll link to https://nats.io/